.github/workflows/gating.yaml

```diff
@@ -46,3 +46,67 @@ jobs:
               echo 'RUN python3 src/ipa-tuura/manage.py test'
           } > podmanfile
           podman build --tag gatingtest -f ./podmanfile
+
+  build-fedora-ipa-server-container-on-ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sssd repository
+        uses: actions/checkout@v3
+        with:
+          path: sssd
+      - name: Setup containers
+        uses: SSSD/sssd-ci-containers/actions/setup@master
+        with:
+          path: sssd-ci-containers
+          tag: latest
+          override: |
+            services:
+              client:
+                image: quay.io/sssd/ci-client-devel:latest
+                shm_size: 4G
+                tmpfs:
+                - /dev/shm
+                volumes:
+                - ../sssd:/sssd:rw
+              keycloak:
+                image: quay.io/ftrivino/keycloak
+                shm_size: 4G
+                tmpfs:
+                - /dev/shm
+                volumes:
+                - ../sssd:/sssd:rw
+
+      - name: Get source
+        uses: actions/checkout@v3
+        with:
+          path: 'ipa-tuura'
+# yamllint disable rule:line-length
+      - name: Create container and run tests
+        run: |
+          {
+              echo 'FROM fedora:38'
+              echo '# set TZ to ensure the test using timestamp'
+              echo 'ENV TZ=Europe/Madrid'
+              echo 'RUN dnf -y update'
+              echo 'RUN dnf -y install python3-sssdconfig maven unzip python3-pip git python3-netifaces python3-devel krb5-devel gcc sssd-dbus dbus-devel glibc glib2-devel dbus-daemon python-devel openldap-devel python-ipalib'
+              echo 'RUN dnf clean all'
+              echo 'COPY ipa-tuura ipa-tuura'
+              echo 'WORKDIR /ipa-tuura'
+              echo 'RUN pip install -r src/install/requirements.txt'
+              echo 'RUN pip install dbus-python python-ldap'
+              echo 'RUN python3 src/ipa-tuura/manage.py test'
+          } > podmanfile
+          podman build --tag gatingtest -f ./podmanfile
+
+      - name: Pull bridge
+        run: |
+          sudo podman pull quay.io/idmops/bridge
+
+      - name: List containers
+        run: |
+          sudo podman ps -a
+          sudo sleep 120
+          sudo podman exec ipa /usr/bin/bash -c 'ipactl restart'
+          sudo sleep 30
+          sudo podman exec ipa /usr/bin/bash -c 'echo Secret123 | kinit admin && echo vagrant'
+          sudo podman exec ipa /usr/bin/bash -c 'klist'
```

Comment:
An example of usage of PR number in actions the `${{github.event.number}}` variable (NOT TESTED with latest changes): 
```yaml
build:
    runs-on: ubuntu-latest
    env:
      IMAGE_TAG: ${{ github.event.number || github.ref_name  }}
    name: build the docker image
    steps:
      - name: Checkout the repository
        uses: actions/checkout@v3

      - name: Set up QEMU for docker
        uses: docker/setup-qemu-action@v2

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2

      - name: Print IMAGE_TAG to be used
        run: echo "${{ env.IMAGE_TAG }}"

      - name: Print SECRETS as **** if they exist
        run: echo "username; ${{ secrets.REGISTRY_USERNAME }}"

      - name: Login to quay.io
        uses: docker/login-action@v2
        with:
          registry: quay.io
          username: ${{ secrets.REGISTRY_USERNAME }}
          password: ${{ secrets.REGISTRY_TOKEN }}

      - name: Build and push the image with a given tag
        uses: docker/build-push-action@v3
        with:
          context: ./src
          file: ./src/Containerfile 
          push: true
          tags: "quay.io/idmops/${{ github.event.repository.name }}:${{github.event.number}}"

  test:
    name: test the built image
    runs-on: ubuntu-latest
    container:
      image: "quay.io/idmops/${{ github.event.repository.name }}:${{ github.event.number || github.ref_name }}"  # here we can probably use GH registry and not quay as pull will not work or push for PRs
    needs: [pre-commit, test-on-ubuntu, build]

    steps:  # inside container
      - name: Display Container OS version
        run: cat /etc/redhat-release || uname -a

      - name: Run basic check for file presence
        run: file /ipa-tuura/manage.py || echo "/ipa-tuura/manage.py not found"

```